.gitignore

```diff
@@ -9,4 +9,9 @@ tools/coronavirus.py
 coronavirus_2020.egg-info
 default.profraw
 tools/generate-individiual-plots.html
+<<<<<<< HEAD
```

Comment:
I think something went wrong while merging the master branch to this one, the diff wasn't properly  sorted out here

Reply:
I have fixed that.